frontend/testing/components/Footer.test.tsx

```diff
@@ -15,6 +18,8 @@ describe("Footer", () => {
 
   it("Basic render functionality", () => {
     const { queryByText } = render(<Footer />);
```

Comment:
I realized this a bit late but -- can we use `renderWrapped` exported from the helpers.tsx file instead of render -- it's because renderWrapped has the appropriate providers wrapped and would give us a better picture on the actual component we're testing!